textual/src/toga_textual/widgets/numberinput.py

```diff
@@ -0,0 +1,84 @@
+from travertino.size import at_least
+
+from textual.validation import Number
+from textual.widgets import Input as TextualInput
+
+from .base import Widget
+
+
+class TogaInput(TextualInput):
+    def __init__(self, impl):
+        super().__init__()
+        self.interface = impl.interface
+        self.impl = impl
+
+    def on_input_changed(self, event: TextualInput.Changed) -> None:
+        self.interface.on_change()
+
+    def on_input_submitted(self, event: TextualInput.Submitted) -> None:
+        self.interface.on_confirm()
```

Comment:
If I press Enter when the phone has focus, I get an exception because Numberinput doesn't support `on_confirm()`